cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,553 @@
+General information on rule format
+==================================
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
```

Comment:
Add : "Enrichment rules enable calculating the value of an attribute according specific conditions. Those rules are regularly applied, as a user who would regularly edit product."